package.json

```diff
@@ -5,6 +5,10 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
+    "mock:server": "npm run concat:mocks && json-server --routes routes.json --watch mock/data.json",
+    "start:proxy": "ng serve --proxy-config proxy.conf.json",
+    "start:proxy:mock:server": "concurrently --kill-others \"npm run mock:server\" \"npm run start:proxy\"",
```

Comment:
would prefer something like `start:mock` or `dev`. Personally typing `"start:proxy:mock:server` seems like a pain.

Reply:
I did updated it and next commit! good suggestion